js/localization/intl/number.js

```diff
@@ -5,6 +5,7 @@ import openXmlCurrencyFormat from '../open_xml_currency_format';
 import accountingFormats from '../cldr-data/accounting_formats';
 
 const CURRENCY_STYLES = ['standard', 'accounting'];
+const MAX_FRACTION_DIGITS = 20;
```

Comment:
Why do we use two constants with the same name and value?

Reply:
Because it's the same spec boundary
Tried to import it from 1 file to another - we have some legacy imports in this code and many tests got failing